ui/apps/platform/src/Containers/Compliance/Dashboard/ComplianceDashboardPage.tsx

```diff
@@ -148,7 +147,6 @@ function ComplianceDashboardPage(): ReactElement {
                         bodyClassName="pr-4 py-1"
                         className="pdf-page"
                     />
-                    <ComplianceByStandards />
```

Comment:
Can you remove `?` optional in props declaration (but not the conditional logic in the component).
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Compliance/widgets/ComplianceByStandards.tsx#L19-L23

The 3 props are required and specified for remaining 4 use cases:
* Cluster.js
* Deployment.js
* Namespace.js
* Node.js